uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestTest.php

```diff
@@ -38,7 +38,7 @@ public function setUp() {
     $this->registerCurrentLoggedInContactInSession($this->leaveContact);
     CRM_Core_Config::singleton()->userPermissionClass->permissions = [];
 
-    $this->leaveRequestStatuses = $this->getLeaveRequestStatuses();
+    $this->leaveRequestStatuses = LeaveRequest::getStatuses();
```

Comment:
Shouldn't this test class be using the LeaveRequestHelpersTrait.php so that the option values are loaded from the xml file?

Reply:
It turns out that this was only really used by a single method, so instead of using the class property, I've change it to load the statuses inside the test